endpoints/oauth/login.py

```diff
@@ -297,6 +301,26 @@ def attach_func():
         if result.error_message is not None:
             return _get_response(result)
 
+        # Conduct RedHat Export Compliance if enabled
+        if features.EXPORT_COMPLIANCE:
+            logger.debug("Attempting to hit export compliance service")
+            result = requests.post(
+                app.config.get("EXPORT_COMPLIANCE_ENDPOINT"),
```

Comment:
@kleesc This is the endpoint, it is set as a config variable which is in the staging config since we don't want to expose it publicly.